src/processing/shape/LineDrawer.java

```diff
@@ -2,6 +2,10 @@
 
 import java.util.ArrayList;
 import processing.utility.*;
+import infrastructure.validation.logger.LoggerFactory;
+import infrastructure.validation.logger.ILogger;
+import infrastructure.validation.logger.LogLevel;
+import infrastructure.validation.logger.ModuleID;
```

Comment:
Add reviewer's name (my name) in this file.

Reply:
ok I'll add now